### DISCUSSION TITLE
How to not truncate exception info?

### DISCUSSION THREAD

**Discussion by ReenigneArcher** [fastapi/typer#879] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorials in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
n/a
```

### Description

* Using [rstcheck](https://github.com/rstcheck/rstcheck)
* When there is invalid rst file, the formatted exceptions are not helpful due to truncating. For example, see https://github.com/rstcheck/rstcheck-core/issues/99, where many places have shortened paths.

e.g.
```txt
| |                 files = [                                               | |
| |                             WindowsPath('C:/Users/frog/Desktop/SunshinÃ  | |
| |                         ]                                               | |
```

I'm not familiar with Typer, but if possible I'd like to know how to configure this behavior so I can submit a PR for rstcheck. I read through the documentation, but actually couldn't find any of the parameters than can be provided to the Typer app/object.

### Operating System

Windows

### Typer Version

0.12.3

### Python Version

3.11

**Comment by svlandeg** [comment]
Hi!

I think this is caused by Rich formatting. If you uninstall Rich in your environment, do you still see the table formatting?

We have an open PR to allow disabling Rich even if it's installed in your env: https://github.com/tiangolo/typer/pull/859

Let me know if that's helpful!

**Reply by ReenigneArcher** [reply]
Sorry for the late reply. I opened this discussion for another developer, on one of my projects, who was facing an issue. I passed the info on to them, but haven't heard back.

**Reply by YuriiMotov** [reply]
So, I think we can close this then.
Either `rich_markup_mode=None` option ([docs](https://typer.tiangolo.com/tutorial/commands/help/#rich-markdown-and-markup)) or options explained in [Disable Pretty Exceptions](https://typer.tiangolo.com/tutorial/exceptions/#disable-pretty-exceptions) should help
